src/env.rs

```diff
@@ -218,6 +218,16 @@ fn fill_dependencies(
             Ok(None) => warn!("Missing URL for {}", dep),
             Err(e) => return Err(e.into()),
         };
+
+        let frontend_key = format!("{}_FRONTEND_URL", dep.replace("-", "_").to_uppercase());
+        match client.get::<String>(&format!("infrastructure/service-uris/{}", frontend_key)) {
+            Ok(Some(val)) => {
+                trace!("Merging to environment: {:?}: {:?}", frontend_key, val);
+                env.insert(frontend_key, val);
+            }
+            Ok(None) => warn!("Frontend URL for {} either not needed or not set", dep),
```

Comment:
This seems to me like maybe it should be an `info` or `debug`, rather than a `warn`, as we do expect it to happen sometimes, and it's not really a problem (is my assumption, tell me if I'm wrong).

Reply:
Good point. I'll make it an info. We definitely expect the frontend url to not be set for services that don't have one, and like you mentioned before, if a service should have a frontend url set and it isn't set for whatever reason, we should know about that too.